src/es/2018-01-jo/04/02.md

```diff
@@ -1,6 +1,26 @@
 ---
-title:  Lección
+title:  Reducir
 date:   21/01/2018
 ---
 
-### <center>Todavía estamos trabajando en esta lección. Por favor, vuelva más tarde.</center>
\ No newline at end of file
+**Evidencia > Santiago 1:121**
+
+Hay un movimiento que se está tornando muy popular en los Estados Unidos, llamado "el movimiento de casas pequeñas”. Las personas que forman parte de este movimiento venden sus casas y construyen o compran una casa nueva que es mucho más pequeña que la anterior. Algunas de estas "casas pequeñas" tienen menos de 65 m2 para una familia de cuatro. Para hacer este cambio, la familia debe vender, donar o deshacerse de gran parte de sus pertenencias. La nueva casa necesita usos creativos del espacio para aprovechar al máximo el espacio y dar lugar a lo que quedó. 
+
+Hay dos cosas principales que este cambio logra. Primero, los miembros de la familia están mucho más cerca unos de otros; y segundo, tienen limitaciones para sus posesiones materiales. A causa de esto, esas familias están descubriendo que pasan más tiempo juntos, y tienen menos deseos de comprar los productos más novedosos y fantásticos, como la última pantalla de televisión, prenda de vestir o mueble. Las familias que permanecen firmes en el cambio descubren que son más felices que cuando vivían en casas grandes. 
+
+Piensa en esto por un minuto. ¿Qué sería limpiar una casa de 185 m2en comparación con una de 65 m2? Cuando lo desglosas en fracciones de la vida diaria, estás obteniendo mucho más tiempo para hacer lo que es importante. Esto también significa que el estrés que experimentas al ocuparte de una casa y todo lo que hay en ella -además de estar al tanto de las modas- desaparece casi por completo. Imagina la paz mental y la libertad que obtienes al manejar muchas menos cosas que no son esenciales. 
+
+Como mayordomos del mundo de Dios, debemos seguir este patrón. Al dejar de lado el deseo de ganancias materiales, nos liberamos para pasar más tiempo con él y con nuestro prójimo. Santiago 1:21 dice: “Por esto, despójense de toda inmundicia y de la maldad que tanto abunda, para que puedan recibir con humildad la palabra sembrada en ustedes, la cual tiene poder para salvarles la vida". A veces, al luchar por obtener el éxito material, perdemos de vista el objetivo final. Es mucho más importante conocer a Dios y servirlo que obtener lujos y comodidades mundanales. 
+
+### Para Pensar y Debatir
```

Comment:
I think doing it bold i.e `**` as opposed to `###` would make it better @clope031